README.md

```diff
@@ -76,6 +76,32 @@ struct A {
 }
 ```
 
+### Enum with explicit discriminant
+
+`#[borsh(use_discriminant=false|true)]` is required if you have an enum with explicit discriminant. This is because the discriminant is not serialized by default, and the deserializer needs to know whether to use the discriminant or not.
```

Comment:
remove `This is because the discriminant is not serialized by default, and the deserializer needs to know whether to use the discriminant or not.` and add an example of enum which raises compilation error with error text in comment.
Add a line, that the attribute affects `BorshSerialize` and `BorshDeserialize` behaviour at the same time.